src/test/java/uk/gov/pay/api/model/PaymentTest.java

```diff
@@ -46,7 +46,7 @@ public void shouldNotPrintEmailAndCardDetailsWhenToString() throws IOException {
                 "  }\n" +
                 "}", ChargeFromResponse.class);
 
-        PaymentWithAllLinks payment = PaymentWithAllLinks.valueOf(paymentFromConnector, selfUri, eventsUri, cancelUri, refundsUri);
+        PaymentWithAllLinks payment = PaymentWithAllLinks.valueOf(paymentFromConnector, selfUri, eventsUri, cancelUri, refundsUri, null);
```

Comment:
In reality, the capture URL isn’t ever going to be `null`, so I think we should probably pass in a dummy `URI` object (like we do for the other URLs, which are created on lines 21 through 24)

Reply:
Looks like this comment was missed